### DISCUSSION TITLE
Manually choosing the train, validation and test sets

### DISCUSSION THREAD

**Discussion by Iheb-prepa** [open-edge-platform/anomalib#2034] [Q&A] [answered]
Hello,
I'd like to express my gratitude for this incredibly valuable library and commend you for your exceptional work.
I'm curious to know if there's a way to manually designate the training, validation, and test sets as input. 
While I've explored the Folder and FolderDataset classes, it seems that they don't fulfill this requirement, as they perform their own data split based on predefined parameters and a seed.
Therefore, I'm wondering if Anomalib features a mechanism that enables me to precisely specify the images for training, validation, and testing. Because I want to do some experiments by varying the images used for training and validation and test.
I'm looking for something equivalent to (if it exists):
Data(train_path =/path/to/train, validation_path=/path/to/validation, test_path=/path/to/test)
Thank you in advance.

**Comment by samet-akcay** [comment, marked as answer]
Hi @Iheb-prepa, this is currently not possible, but on our roadmap to enable this. @djdameln adding you for visibility

**Reply by Iheb-prepa** [reply]
Thank you for your response.

**Reply by Iheb-prepa** [reply]
Hi, is it possible to know the approximate date of release of this feature? 
Thanks in advance.

**Reply by Iheb-prepa** [reply]
@samet-akcay Hi, is it possible to know the approximate release date of this feature please? Is it going to be soon or in the next months or later than that? Thanks in advance.

**Reply by samet-akcay** [reply]
@Iheb-prepa it would be within the next few months as the entire team are occupied with some other tasks. Probably in v1.2 (end of july)

**Reply by Iheb-prepa** [reply]
Thanks a lot!

**Comment by ckjunaid** [comment]
I seen a code for downloading the data set using wget .. I will update u the further and detailed code soon

**Comment by samet-akcay** [comment]
There is a PR addressing this #2239
